docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,166 @@
+- [What to review](#what-to-review)
+- [Preparing code for review](#preparing-code-for-review)
+- [Commit messages](#commit-messages)
+- [Performing code reviews](#performing-code-reviews)
+- [Responding to reviews](#responding-to-reviews)
+- [Examples](#examples)
+
+
+## Motivation
+
+We perform code reviews (CRs) in order to improve code quality and benefit from
+[positive effects on team and company
+culture](https://blog.fullstory.com/what-we-learned-from-google-code-reviews-arent-just-for-catching-bugs).
+For example:
+
+- **Committers are motivated** by the notion of a set
   of reviewers who will look over the change request: the committer tends to
   clean up loose ends, consolidate TODOs, and generally improve the commit.
+  Recognition of coding expertise through peers is a source of pride 
+  for many programmers.
 
-- **Sharing knowledge** helps development teams in at least three
-  ways:
+- **Sharing knowledge** helps development teams in several ways:
 
   - A CR explicitly communicates added/altered/removed functionality to
     team members who can subsequently build on the work done.
 
   - The committer may use a technique or algorithm that reviewers can
-    learn from.
+    learn from. More generally, code reviews help [raise the quality bar across the organization](https://blog.fullstory.com/what-we-learned-from-google-code-reviews-arent-just-for-catching-bugs/).
 
-  - Reviewers may possess knowledge about the code base that can help
+  - Reviewers may possess knowledge about programming techniques or the code base that can help
     improve or consolidate the change; for example, someone else may be
     concurrently working on a similar feature or fix.
 
+  - Positive Interaction and communication strengthens social bonds between
+    team members.
+
 - **Consistency** in a code base makes code easier to read and
   understand, helps prevent bugs, and facilitates collaboration
   between regular and migratory developer species.
 
 - **Legibility** of code fragments is hard to judge for the author
-  whose brainchild it is, and easy to judge for a reviewer who does
+  whose brain child it is, and easy to judge for a reviewer who does
   not have the full context. Legible code is more reusable, bug-free,
   and future-proof.
 
 - **Accidental errors** (e.g., typos) as well as **structural errors**
-  (e.g., dead code, logic or algorithm bugs) are often much easier to
-  spot for critical reviewers with an outside perspective. (Often,
-  it's a rather satisfying sport to find bugs...)
+  (e.g., dead code, logic or algorithm bugs, performance or architecture
+  concerns) are often much easier to
+  spot for critical reviewers with an outside perspective.
+  Studies have found that even short and informal code reviews have significant
+  [impact on code quality and bug frequency](https://blog.codinghorror.com/code-reviews-just-do-it/).
+
+- **Security reviews** are required in some environments. They are performed
```

Comment:
I would welcome a relevant external link here to explain appsec reviews, if you have one handy. If not, no big deal.

Reply:
added